README.md

```diff
@@ -120,37 +120,37 @@ See individual pages for details!
 + MF = "multfield" baseline
```

Comment:
hey @amyxie361 there's a typo I just noticed here - "multifield" - missing "i", can you please fix?